extensions/wikia/Answers/AttributionCache.class.php

```diff
@@ -250,79 +277,83 @@ private function sortContribs(Array $contribs) {
 	 * @param int $userId user id
 	 * @param int $pageId page id for calculating edits per article
 	 */
-	private function rebuildUserEditPoints($userId, $pageId = 0) {
+	private function rebuildUserEditPoints( $userId, $pageId = 0 ) {
 		global $wgAnswerHelperIDs, $wgMemc;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgAnswerHelperIDs" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgMemc" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app